frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx

```diff
@@ -207,7 +207,7 @@ export const CreateOperandForm: React.FC<CreateOperandFormProps> = (props) => {
       }
       const schemaPath = field.path.split('.').join('.properties.');
       const required = (_.get(
-        props.openAPI,
+        props.openAPI.properties,
```

Comment:
Is `props.openAPI` ever undefined?

Reply:
@spadgett The `props.openAPI`should not be `undefined`, since it is parsed from the CSV, but you are right, we should have the error handling logic here.